src/main/resources/templates/fragments/layout.html

```diff
@@ -62,6 +62,13 @@
                           <span>List of Owners</span>
                       </li>
                 </th:block>
+				
+				<th:block th:if="${toggles[2] == true}">
```

Comment:
You can remove this block of added code since it would simply make a copied navbarm menu item, and not effect the actual location search box